kyo-scheduler/jvm/src/main/scala/kyo/scheduler/package.scala

```diff
@@ -0,0 +1,10 @@
+package kyo
+
+package object scheduler:
+
+    private def statsScope(path: String*) =
+        "kyo" :: "scheduler" :: path.toList
+
+    private def bug(msg: String, ex: Throwable) =
+        (new Exception("🙈 !!Kyo Scheduler Bug!! " + msg, ex)).printStackTrace(System.err)
```

Comment:
```suggestion
    import java.util.logging.*
    private lazy val logger = Logger.getLogger(getClass.getName)
    
    private def bug(msg: String, ex: Throwable) =
        logger.log(Level.SEVERE, "🙈 !!Kyo Scheduler Bug!!", new Exception(msg, ex))
```
Since you want 0 dependencies, we can still use java.util.logging, since that's part of the standard library. This will give people better interoperability with other logging frameworks or other tools hooked on logging, like Sentry.

Reply:
good idea! 🙏

Reply:
You can define your own logging and then there will be a adapter. 

Netty 5 is using slf4j again